### ISSUE TITLE
Merge

### ISSUE THREAD

**Issue by mtchibozo** [CharlesWu123/SPLERGE#2]
Thank you for implementing this so neatly.
Two questions: Was the model you provided trained only with split? or with split+merge (in the loss function file, the merge loss is commented and is not taken into account)?
Is there a version of the code where train.py an test.py include the merging of cells? As it is now, the code only splits an image.
Thank you

**Comment by CharlesWu123** [comment]
1. Yes, only split. the merge loss is commented, because the function get_merge_input() is not implement completely in loss.py (If you read the paper, you can implement it).
2. Yes, only split, not merge. Due to work, did not continue to do.
3. If you are interested, you can continue to implement it .

**Comment by mtchibozo** [comment]
Thanks Charles! I will 👍
